po/es.po

```diff
@@ -437,8 +437,8 @@ msgid ""
 "By <a href=\"https://wiki.gnucash.org/wiki/Development\">developing the "
 "source code</a>"
 msgstr ""
-"A través del<a href=\"https://wiki.gnucash.org/wiki/Development\">desarrollo "
-"del código fuente</a>"
+"Al <a href=\"https://wiki.gnucash.org/wiki/Development\">desarrollar del "
```

Comment:
@gallegonovato `Development\">desarrollar` no space?

Reply:
Good afternoon @fellen

What is this string https://hosted.weblate.org/translate/gnucash/website/es/?checksum=0d8563195c8f0700 ?

I don't understand the space you are talking about?

Best regards

Reply:
2.6-release-tour.phtml is no longer reachable over the menu, but you can enter it in your browser as
https://www.gnucash.org/2.6-release-tour.phtml
But I was wrong about the space, sorry.

Reply:
Good afternoon:

You don't have to apologize for anything.

That I may miss a space when writing, and not realize it. 

But I didn't understand about the space in this case.

A hug

Reply:
Buenas tardes!

At first I thought, it was the closing tag of a link. Then usually a space follows or a sentence sign.
But it is the opening tag and then no space follows.